### PR TITLE
Update to version 5.1.4

### PR DESCRIPTION
**Pull request by 183amir**
It would be a good idea to know the API/ABI compatibility before we merge this in.

### CODE REVIEW

**Comment by 183amir** [comment]
Merging this will fix this nasty bug in `bob.io.image` that we have seen: https://gitlab.idiap.ch/bob/bob.io.image/issues/13

**Comment by 183amir** [comment]
This seems to be broken because of this issue: https://github.com/conda/conda-build/issues/1248
So I think when we move to conda-build 2 this should get fixed by itself.

**Comment by 183amir** [comment]
I think we are blocked by this: https://github.com/conda-forge/conda-forge-build-setup-feedstock/pull/33 and use of the old conda-build was introduced here: https://github.com/conda-forge/conda-forge-build-setup-feedstock/pull/27

**Comment by ocefpaf** [comment]
@183amir if you are in a hurry to get this in the channel b/c of the bug you can either by-pass the `conda-build` version like [this](https://github.com/conda-forge/pyferret-feedstock/commit/bbaecb8925b05f19322002540314f860453b6395) or you can comment out the `conda inspect` for now.

**Comment by 183amir** [comment]
No I think this pull will take time because it is likely that API will break: http://upstream.rosalinux.ru/versions/giflib.html so we have to pin packages first before merging this.

**Comment by ocefpaf** [comment]
> No I think this pull will take time because it is likely that API will break: http://upstream.rosalinux.ru/versions/giflib.html

I do not see anything above `5.1.1` there.

> so we have to pin packages first before merging this.

It is up to you. I am OK trusting that this minor release does not include any breakages. I also test it on Linux for `gdal` and things are smooth there.

**Comment by 183amir** [comment]
> I do not see anything above 5.1.1 there.

Their history shows that they don't follow semantic versioning so I think we should follow a pin on the patch version.

I have asked @lvc to add `giflib` to http://abi-laboratory.pro/tracker if possible, let's see what will happen in that analysis.

**Comment by 183amir** [comment]
Hi @ocefpaf, could you please merge this at your earliest convenience? Thanks.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -22,7 +22,7 @@ test:
         - giffix -h
         - gifinto -h
         - giftext -h
-        - conda inspect linkages giflib --name _test  # [linux]
+        # - conda inspect linkages giflib --prefix ${CONDA_DEFAULT_ENV}  # [linux]
```
Comment (ocefpaf) [inline recipe/meta.yaml:25]: This will work as soon as we start using `conda-build 2`.

**Pull request merged**